stackit/internal/services/mariadb/instance/resource.go

```diff
@@ -124,6 +138,17 @@ func (r *instanceResource) Schema(_ context.Context, _ resource.SchemaRequest, r
 		"plan_id":     "The selected plan ID.",
 	}
 
+	parametersDescriptions := map[string]string{
+		"sgw_acl":                "Comma separated list of IP networks in CIDR notation which are allowed to access this instance.",
+		"graphite":               "Graphite server URL (host and port). If set, monitoring with Graphite will be enabled.",
+		"enable_monitoring":      "Enable monitoring.",
+		"max_disk_threshold":     "The maximum disk threshold in MB. If the disk usage exceeds this threshold, the instance will be stopped.",
+		"metrics_frequency":      "The frequency in seconds at which metrics are emitted.",
+		"metrics_prefix":         "The prefix for the metrics. Could be useful when using Graphite monitoring to prefix the metrics with a certain value, like an API key",
+		"monitoring_instance_id": "The ID of the monitoring instance (e.g. Argus instance). Required if `enable_monitoring` is set to `true`.",
```

Comment:
Sorry to be picky here, but this can **only** be a STACKIT monitoring instance, right? 

I also tested manually and actually the `monitoring_instance_id` is not required if `enable_monitoring` is `true`, somehow they automatically set an existing Argus instance in the project to be the monitoring if the `id` is not provided. (tested with Redis for speed)

Reply:
Good point, I was thinking os the LB which allows observability via external solutions, but in that case they expect a push URL. In this case since this is just an instance ID it probably expects a STACKIT instance ID

Reply:
I am testing the automatic set of the monitoring instance ID, I though it failed before for me on MariaDB

Reply:
I think it has failed for me as well in the past

Reply:
It is working as you described. I also think you should go into too much details then if the behaviour is not described anywhere yet